templates/fullstack/.graphqlrc.yml

```diff
@@ -34,24 +34,18 @@ extensions:
     crud:
       create: true
       update: true
-      findAll: true
       find: true
+      findOne: true
       delete: true
       subCreate: true
       subUpdate: true
       subDelete: true
     ## Codegen plugins
     plugins:
         graphback-schema:
-          format: 'graphql'
           outputPath: ./server/src/schema
```

Comment:
Recommended practice is full path to output schema.

Reply:
Good catch!